x/bounty/types/msgs.go

```diff
@@ -1,15 +1,18 @@
 package types
```

Comment:
Why changed the file name from msg.go to msgs.go?

Reply:
This file includes MsgSubmitFinding and MsgCreateProgram